airbyte-webapp/src/components/connection/CatalogTree/styles.tsx

```diff
@@ -20,3 +20,13 @@ export const ArrowCell = styled(HeaderCell)`
 export const NameContainer = styled.span`
   padding-left: 30px;
 `;
+
+export const SyncHeaderContainer = styled.span`
+  padding-left: 34px;
+  display: inline-block;
+  min-width: 62px;
+`;
+
+export const SyncCheckboxContainer = styled.div`
+  padding-left: 24px;
+`;
```

Comment:
Kind of nasty hard-coded values 😕 but since this table won't be around long anyway, I did not spend much time here.